### PR TITLE
feat: [CDS-69156]:fs activity logs

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
Activity logs list for file store

#### Screenshots
Screenshot 2023-05-22 at 15 15 04

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger coverage

**Comment by litvinovstas** [comment]
retrigger jest

**Comment by litvinovstas** [comment]
retrigger cypress-cv

**Review by satyam-harness — APPROVED** [review]

File: src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.table {
```
Comment (Sagar-Rajput7) [inline src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss:1]: NIT: Autogenerated polyform licenses are not present for all of these newly added files

File: src/modules/31-filestore/components/ActivityLog/FileStoreLogs.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+import { Container, Layout, Popover, Button, Text, PageSpinner } from '@harness/uicore'
+import { useStrings } from 'framework/strings'
+
+import { useListActivities } from 'services/cd-ng'
+import { ActivityLogFileStoreList } from './ActivityLogFileStoreList'
+import { FileStoreContext } from '../FileStoreContext/FileStoreContext'
+
+import css from './ActivityLogFileStore.module.scss'
+
+const DATE_FORMAT = 'DD MMMM YYYY '
```
Comment (Sagar-Rajput7) [inline src/modules/31-filestore/components/ActivityLog/FileStoreLogs.tsx:16]: NIT: extra space at the end, and also we can export this to use at other places in this PR.

File: src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+}
+
+.dateRangePicker {
+  padding: var(--spacing-large);
+}
+.popoverClassname {
+  border-radius: 4px;
```
Comment (Sagar-Rajput7) [inline src/modules/31-filestore/components/ActivityLog/ActivityLogFileStore.module.scss:15]: NIT: 
```suggestion
  border-radius: var(--spacing-2);
```

**Review by Sagar-Rajput7 — APPROVED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

**Review by satyam-harness — APPROVED** [review]

**Review by YounixIO — APPROVED** [review]

**Pull request merged**